### ISSUE TITLE
Player models are not visible in multi-player

### ISSUE THREAD

**Issue by skaldarnar** [Terasology/CoreRendering#11]
The cube player models are no longer visible in multi-player. Tools in hand render fine, and light sources held in hand will emit light.

![image](https://user-images.githubusercontent.com/1448874/83336432-9c19be80-a2b3-11ea-83a8-f4f5f266f533.png)

**Comment by AndyTechGuy** [comment]
This issue is not exclusive to player models in multiplayer, in single player NPCs are no longer visible:

![image](https://user-images.githubusercontent.com/44126397/83359321-1f8aec80-a354-11ea-8e54-b976a92a61b3.png)
_From Terasology/MetalRenegades: The grey shopkeeper character is invisible, but can be seen on the upper tooltip & minimap._

One discovery I made while trying to get this screenshot: somehow the character models ARE visible when the player is standing inside of the NPC:

![image](https://user-images.githubusercontent.com/44126397/83359409-a4760600-a354-11ea-82a6-abb36dadb031.png)

**Comment by skaldarnar** [comment]
While talking with @jdrueckert and @AndyTechGuy about this issue we got the idea that this might be related to something related to **inversion** (models being "inside-out" and then getting optimized away of sorts?).

I also had this weird effect where the world seemed to be "upside down", and parts of the world just not getting rendered. Suddenly feels related to this issue...
![image](https://user-images.githubusercontent.com/1448874/83438891-70790e80-a442-11ea-8f1c-2b554e8c75a2.png)

@pollend might this be related to some JOML (conversion) bug? There was something about matrices needed to be inverted or something, wasn't it?

**Comment by pollend** [comment]
that's strange, wonder if it is another cache problem in joml.  any way to easily reproduce this problem.

**Comment by Cervator** [comment]
Two different issues. We just found and fixed the invisible critters bug. Landscape going funky I've seen before, like last play test (happened to me just now once and like in the playtest I couldn't move into the "broken chunk"). I've considered that more part of the batch of weirdness we fixed most of about a month ago - and it seems like there's one left, or it is some rarer older issue more about bad chunks / those kinds of meshes than corrupt rendering.

Progress 🎉
